### PR TITLE
DEV-24 Implements sign up route and controller function

### PR DESCRIPTION
**Pull request by gunnmgirl**
**What kind of change does this PR introduce? (Bugfix, feature, docs update, ...)**
Feature
**What is the current behavior? (You can also link to an open issue here)**
No signup route 
**What is the new behavior (if this is a feature change)?**
Added signup route and controller function for that route 
**Does this PR introduce a breaking change?**
No
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] You’ve rebased your work on the current state of the master development tree
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Your changes follow to the coding standards
- [ ] Your code is documented

**Other Information**:

### CODE REVIEW

File: src/utils/cloudinary.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const cloudinary = require("cloudinary").v2;
```
Comment (Sholja) [inline src/utils/cloudinary.js:1]: - Let us move the integration part to a separate file. We will use module wrapping, make our module for upload, and then export from this module function `upload`. As we talked about this before, when you made axios wrapper, it's much easier tomorrow to replace the module, if needed.

**Review by Sholja — CHANGES_REQUESTED** [review]
- Looks good, please just take a look comment I added about the Cloudinary integration.

File: src/utils/cloudinary.js (hunk 1 above)
Comment (gunnmgirl) [reply src/utils/cloudinary.js:1]: Made a new file imageUploader in utils folder and exported my upload function from there

**Review by Sholja — APPROVED** [review]
Good job!

**Pull request merged**